README.md

```diff
@@ -19,16 +27,30 @@ If you want to update the dependencies to a specific Polkadot SDK version, you c
 ```sh
 # Go to the directory containing the Cargo.toml file you want to update
 cd <cargo-toml-dir>
-# Run the psvm command to update the dependencies
+# Run the psvm command to update the dependencies using PSVM local versions
 psvm
 # You can also update it using the path to the Cargo.toml file
 psvm -p <cargo-toml-dir>/Cargo.toml
 # Overwrite local dependencies with crates.io versions
 psvm -o
-# Update to a specific Polkadot SDK version (default 1.3.0)
+# Update to a specific Polkadot SDK version (default 1.3.0) using PSVM local versions
 psvm -v "1.7.0"
+# Update to the versions defined in a specific Polkadot SDK branch Plan.toml file
+# i.e https://raw.githubusercontent.com/paritytech/polkadot-sdk/release-crates-io-v1.6.0/Plan.toml
+psvm -b "release-crates-io-v1.6.0"
+# Update to the versions defined in a specific Polkadot SDK branch Cargo.lock file
+# i.e https://raw.githubusercontent.com/paritytech/polkadot-sdk/release-crates-io-v1.3.0/Cargo.lock
+psvm -b "release-crates-io-v1.3.0" -s "Cargo.lock"
 ```
 
+### Offline
+
+You can use PSVM offline with the `-v` or `--version` flag, cause PSVM keeps a local mapping (i.e. [v1.3.0 Mapping](/src/versions/release-crates-io-v1.3.0.json)) of the Polkadot SDK dependencies versions, that will be slowly updated. So be aware that the latest Polkadot SDK versions may not be available in the local mapping.
```

Comment:
Sure that we need the offline version at all? As soon as there are two ways to do something and one is wrong, then ppl will end up doing the wrong thing and complain.

Reply:
I keep it to just not cause a breaking change with the "offline" justification, but probably is useless cause anyways you need internet to download the crates after setting a version.

Reply:
The only concern here is the fact that some branches don't have a Plan.toml file for example. What do you about fallback to the Cargo.lock in case Plan.toml not found. For example v1.3.0: https://raw.githubusercontent.com/paritytech/polkadot-sdk/release-crates-io-v1.3.0/Plan.toml

Reply:
Ok! It's done, also added a test to check if it works for all release branches, and it works! The only branch without Plan.toml is v1.3.0, but in this case it will fallback to Cargo.lock. Added a warning in the Workflow section.